pkg/api/server.go

```diff
@@ -245,38 +249,32 @@ func Start(uiBox embed.FS, loginUIBox embed.FS) {
 		TLSConfig: tlsConfig,
 	}
 
-	go func() {
-		printVersion()
-		printLatestVersion(context.TODO())
-		logger.Infof("stash is listening on " + address)
-		if tlsConfig != nil {
-			displayAddress = "https://" + displayAddress + "/"
-		} else {
-			displayAddress = "http://" + displayAddress + "/"
-		}
-
-		// This can be done before actually starting the server, as modern browsers will
-		// automatically reload the page if a local port is closed at page load and then opened.
-		if !c.GetNoBrowser() && manager.GetInstance().IsDesktop() {
-			err = browser.OpenURL(displayAddress)
-			if err != nil {
-				logger.Error("Could not open browser: " + err.Error())
-			}
-		}
+	printVersion()
+	printLatestVersion(context.TODO())
```

Comment:
this was run in a go function so that we dont need to wait for the response/timeout/network error and thus delay the start

Reply:
Fixed in latest